docs/Create.md

```diff
@@ -0,0 +1,460 @@
+
+You can customize the redirection by setting the `redirect` prop to one of the following values:
+
+- `'edit'`: redirect to the Edit view (the default)
+- `'list'`: redirect to the List view
+- `'show'`: redirect to the Show view
+- `false`: do not redirect
+
+```jsx
+const PostCreate = () => (
+    <Create redirect="list">
+        ...
+    </Create>
+);
+```
+
+Note that the `redirect` prop is ignored if you set [the `mutationOptions` prop](#mutationoptions). See that prop for how to set a different redirection path in that case. 
+
+If you want to allow the user to enter several records one after the other, setting `redirect` to `false` won't make it, as the form isn't emptied by default. You'll have to empty the form using the `mutationOptions`, and this option disables the `redirect` prop. Check [the Save And Add Another section](#save-and-add-another) for more details.
+
+## `resource`
+
+Components based on `<Create>` are often used as `<Resource create>` props, and therefore rendered when the URL matches `/[resource]/create`. The `<Create>` component generates a call to `dataProvider.create()` using the resource name from the URL by default.
+
+You can decide to use a `<Create>` component in another path, or embedded in a page using another resource name (e.g. in a Dialog). In that case, you can explicitly set the `resource` name:
+
+```jsx
+const PostCreate = () => (
+    <Create resource="posts">
+        ...
+    </Create>
+);
+```
+
+## `sx`: CSS API
+
+The `<Create>` components accept the usual `className` prop, but you can override many class names injected to the inner components by React-admin thanks to the `sx` property (as most MUI components, see their [documentation about it](https://mui.com/customization/how-to-customize/#overriding-nested-component-styles)). This property accepts the following keys:
+
+| Rule name               | Description                                                                          |
+|-------------------------|--------------------------------------------------------------------------------------|
+| `& .RaCreate-main`      | Applied to the main container                                                        |
+| `& .RaCreate-noActions` | Applied to the main container when `actions` prop is `false`                         |
+| `& .RaCreate-card`      | Applied to the child component inside the main container (MUI's `Card` by default)   |
+
+To override the style of all instances of `<Create>` components using the [MUI style overrides](https://mui.com/customization/globals/#css), use the `RaCreate` key.
+
+## `title`
+
+By default, the title for the `Create` view is "Create [resource_name]".
+
+You can customize this title by specifying a custom `title` prop:
+
+```jsx
+export const PostCreate = () => (
+    <Create title="New post">
+        ...
+    </Create>
+);
+```
+
+The `title` value can be a string or a React element.
```

Comment:
Might be worth reminding users they can access the record that was passed by calling `useRecordContext`.

Reply:
in the context of the Create page, it doesn't make much sense IMHO